packages/devtools_test/.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"url_launcher","path":"/Users/mragonese/Documents/ExternalSDKs/flutter/.pub-cache/hosted/pub.dartlang.org/url_launcher-5.7.10/","dependencies":[]}],"android":[{"name":"url_launcher","path":"/Users/mragonese/Documents/ExternalSDKs/flutter/.pub-cache/hosted/pub.dartlang.org/url_launcher-5.7.10/","dependencies":[]}],"macos":[{"name":"file_selector_macos","path":"/Users/mragonese/Documents/ExternalSDKs/flutter/.pub-cache/hosted/pub.dartlang.org/file_selector_macos-0.0.4+1/","dependencies":[]},{"name":"url_launcher_macos","path":"/Users/mragonese/Documents/ExternalSDKs/flutter/.pub-cache/hosted/pub.dartlang.org/url_launcher_macos-0.0.1+9/","dependencies":[]}],"linux":[{"name":"file_selector_linux","path":"/Users/mragonese/Documents/ExternalSDKs/flutter/.pub-cache/hosted/pub.dartlang.org/file_selector_linux-0.0.2+1/","dependencies":[]},{"name":"url_launcher_linux","path":"/Users/mragonese/Documents/ExternalSDKs/flutter/.pub-cache/hosted/pub.dartlang.org/url_launcher_linux-0.0.1+4/","dependencies":[]}],"windows":[{"name":"file_selector_windows","path":"/Users/mragonese/Documents/ExternalSDKs/flutter/.pub-cache/hosted/pub.dartlang.org/file_selector_windows-0.0.2+1/","dependencies":[]},{"name":"url_launcher_windows","path":"/Users/mragonese/Documents/ExternalSDKs/flutter/.pub-cache/hosted/pub.dartlang.org/url_launcher_windows-0.0.1+3/","dependencies":[]}],"web":[{"name":"file_selector_web","path":"/Users/mragonese/Documents/ExternalSDKs/flutter/.pub-cache/hosted/pub.dartlang.org/file_selector_web-0.8.1+2/","dependencies":[]},{"name":"url_launcher_web","path":"/Users/mragonese/Documents/ExternalSDKs/flutter/.pub-cache/hosted/pub.dartlang.org/url_launcher_web-0.1.5+3/","dependencies":[]}]},"dependencyGraph":[{"name":"file_selector","dependencies":["file_selector_web"]},{"name":"file_selector_linux","dependencies":[]},{"name":"file_selector_macos","dependencies":[]},{"name":"file_selector_web","dependencies":[]},{"name":"file_selector_windows","dependencies":[]},{"name":"url_launcher","dependencies":["url_launcher_web","url_launcher_linux","url_launcher_macos","url_launcher_windows"]},{"name":"url_launcher_linux","dependencies":[]},{"name":"url_launcher_macos","dependencies":[]},{"name":"url_launcher_web","dependencies":[]},{"name":"url_launcher_windows","dependencies":[]}],"date_created":"2021-09-28 14:01:53.696333","version":"2.5.1"}
```

Comment:
this file should not be checked in. There is an example of adding this to the .gitignore file under devtools_app/.gitignore

Reply:
Whoops, thanks for catching! Removed.